.github/workflows/ci.yml

```diff
@@ -81,7 +81,7 @@ jobs:
           function _make() { make -j$(nproc) -s O=out $*; }
           function download_plug_and_trust() { mkdir -p $HOME/se050 && git clone --single-branch -b v0.4.2 https://github.com/foundriesio/plug-and-trust $HOME/se050/plug-and-trust || (rm -rf $HOME/se050 ; echo Nervermind); }
 
-          function download_scp_firmware() { git clone --depth 1 https://github.com/ARM-software/SCP-firmware.git -b v2.13.0 $HOME/scp-firmware || echo Nervermind; }
+          function download_scp_firmware() { git clone --depth 1 https://git.gitlab.arm.com/firmware/SCP-firmware.git -b vg/add-optee-makefile $HOME/scp-firmware || echo Nervermind; }
```

Comment:
Your merge request branch has been destroyed in SCP-firwmare repository (as all other already merge request branches), leaving only the commit SHA1 as reference to where we'd like to sync on.
FYI, I think next SCP-firmware release tag (v2.15.0) is planned around september.